Assets/Scripts/FieldElementsDisabler.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FieldElementsDisabler : MonoBehaviour
+{
+    [SerializeField] private List<GameObject> _circles;
+
+    [SerializeField] private List<GameObject> _crosses;
+
+    private const int NumberOfFields = 9;
```

Comment:
this is a classic mistake. You have same const in DrawDetector and if field size will change you will very likely make a mistake by forgetting to change it somewhere. Same values should be stored in one place